### DISCUSSION TITLE
Revue as an alternative to MailChimp

### DISCUSSION THREAD

**Discussion by leewynne** [forem/forem#16245] [Feature Requests]
Recently came across Revue when I spotted the native integration on a Twitter profile.

It seems light, easy to use and is natively integrated into Twitter. Being able to connect your Forem's RSS feed and then drag articles in is very slick.

Just putting this out there as a potential addition to using MailChimp due to it's native Twitter integration.

https://www.getrevue.co/

**Comment by citizen428** [comment]
[Revue API](https://www.getrevue.co/api#get-/v2/lists) for anyone who wants to look into this.
